src/Ormolu/Imports.hs

```diff
@@ -30,6 +30,7 @@ normalizeImports :: [LImportDecl GhcPs] -> [LImportDecl GhcPs]
 normalizeImports =
   fmap snd
     . M.toAscList
+    . M.mapKeys OrdByPkgQualImportId
```

Comment:
One could also reuse the `Ord` instance for tuples to signal the "prioritization" of `importPkgQual`, which would not require a `newtype`:
```suggestion
    . M.mapKeys (\i -> (importPkgQual i, i))
```
Or even more direct: Just swap `importPkgQual` and `importIdName` in the definition of `ImportId`, such that the derived `Ord` instance does the work for us.

Reply:
This is a great suggestion, thank you! I've gone with the tuple `Ord` suggestion rather than moving `importPkgQual` because it seemed a little more explicit to me. That said, I'm fairly new to writing Haskell so if relying on the ordering of record definitions is idiomatic then let me know & I'll be happy to change it.

Reply:
Yeah, it is certainly more implicit, but upon second look, I think we actually already make use of the field ordering: We have `importIsPrelude :: Bool` as the first field, such that i.e.
```haskell
import A
import Z
import Prelude
```
is a fixed point. If we want to keep that, it would make sense to adapt the field ordering. @mrkkrp WDYT?

Reply:
Ah, well if we're already relying on it I guess there's no harm in continuing to do so. I'll update the PR now.